configuration.yaml

```diff
@@ -72,3 +72,15 @@ TargetSafety:
   toxcast: gs://otar001-core/TargetSafety/data_files/toxcast/ToxCast_2021-08-17.tsv
   aopwiki: gs://otar001-core/TargetSafety/data_files/aopwiki/aopwiki-2023-01-20.json
   outputBucket: gs://otar001-core/TargetSafety/json
+OT_CRISPR:
+  data_directory: gs://otar013-ppp/ot_crispr/input_data
+  config: gs://otar013-ppp/PPP-evidencie-configuration/ot_crispr_config.json
```

Comment:
There's a typo: "evidencie" instead of "evidence". (The GS folder also has this typo, so it should be renamed both there and in the PR)

Reply:
Oh, I can't believe I'm making mistakes like this. Thanks for spotting!